toponetx/classes/cell_complex.py

```diff
@@ -270,7 +272,7 @@ def __getitem__(self, node):
         """
         return self.neighbors(node)
 
-    def _insert_cell(self, cell, **attr):
+    def _insert_cell(self, cell: tuple | list | Cell, **attr):
```

Comment:
TopoNetX is Python 3.9+, so you cannot use union types using the pipe character (Python 3.10+ only). Use `typing.Union`.